modular_zubbers/code/modules/wizard_dize/wizard_dice_object.dm

```diff
@@ -0,0 +1,160 @@
+			user.emote("scream")
+			user.investigate_log("has been smited by a wizardly die of fate.", INVESTIGATE_DEATHS)
+			apply_random_smite(user)
+
+		if(do_teleport)
+			var/turf/desired_turf = get_safe_random_station_turf()
+			if(!desired_turf) //This should never happen, but you never know.
+				current_turf.visible_message(span_userdanger("[src] is erased from reality! Darn!"))
+				qdel(src)
+				return
+
+
+			current_turf.visible_message(span_warning("[src] phases out to another location!"))
+
+			//Teleport!
+			src.forceMove(desired_turf)
+
+			new/obj/effect/temp_visual/emp/pulse(desired_turf) //Does not cause an EMP :^)
+			playsound(desired_turf,'sound/magic/magic_missile.ogg',50,8,FALSE)
+
+			notify_ghosts(
+				"[src] has teleported to [desired_turf.loc]!",
+				source = src
+			)
+
+		return
+
+	current_turf.visible_message(span_userdanger("Magic flows out of [src] and into [user]!"))
+	user.mind.make_wizard()
+
+	qdel(src)
+
+
+/obj/item/dice/d20/teleporting_die_of_fate/proc/apply_random_smite(var/mob/living/target)
+
+	if(!target.client)
+		//Cheater.
+		target.gib(DROP_ALL_REMAINS)
+		return
+
+	var/datum/smite/chosen_smite
+
+	//Unused smites and reasons:
+	// /datum/smite/bsa, can gib
+	// /datum/smite/dock_pay, not coded properly
+	// /datum/smite/fake_bwoink, too minor
+	// /datum/smite/fat, too minor
+	// /datum/smite/ghost_control, too permanent
+	// /datum/smite/gib, hmmmm
+	// /datum/smite/custom_imaginary_friend, requires ghost polling
+	// /datum/smite/immerse, too permanent
+	// /datum/smite/knot_shoes, too minor
+	// /datum/smite/nugget, too serious
+	// /datum/smite/petrify, too permanent
+	// /datum/smite/puzzgrid, too permanent. have you seen the puzzle-solving IQ of bubberstation players????
+	// /datum/smite/puzzle, see above
+	// /datum/smite/scarify, too minor
+	// /datum/smite/supply_pod, /datum/smite/supply_pod_quick is better
+	// /datum/smite/bloodless, way too loud.
+	// /datum/smite/boneless, way too loud
+
+	switch(smite_rng_seed % 9)
+		if(0)
+			var/datum/smite/bad_luck/bad_luck_smite = new
+			bad_luck_smite.silent = FALSE
+			bad_luck_smite.incidents = 7 //7 years bad luck (7 instances)
+			chosen_smite = bad_luck_smite
+		if(1)
+			var/datum/smite/berforate/perforate_smite = new
+			perforate_smite.hatred = "A little" //Actually unironically has a chance to gib.
+			chosen_smite = perforate_smite
+		if(2)
+			chosen_smite = new /datum/smite/brain_damage
+		if(3)
+			var/datum/smite/curse_of_babel/babel_smite = new
+			babel_smite.duration = 5 MINUTES
+			chosen_smite = babel_smite
+		if(4)
+			chosen_smite = new /datum/smite/fireball
+		if(5)
+			chosen_smite = new /datum/smite/lightning
+		if(6)
+			chosen_smite = new /datum/smite/ocky_icky
+		if(7)
+			var/datum/smite/rod/rod_smite = new
+			rod_smite.force_looping = FALSE
+			chosen_smite = rod_smite
+		if(8)
+			var/datum/smite/supply_pod_quick/supply_pod_smite = new
+			supply_pod_smite.target_path = /obj/item/toy/plush/lizard_plushie
```

Comment:
not a bad idea to have a chanec for it to just be harmless, but 

by having a 100% chance for it to have a lil liz plush you remove the suspense from the droppod's dropping, otherwise you could make the guy fear for their fucking life as it approaches

i suggest converting this to a pickweight table with various dangerous things in it, ex. syndicate stormtrooper, a turret, a bomb, etc. with a CHANCE for it to have a liz plush

Reply:
The droppod actually does a singificant amount of damage. It's not the contents, but the damage of the drop pod.